### PR TITLE
Styling updates

### PR DESCRIPTION
**Pull request by DonWillems**
Range of styling updates

Small styling changes:
- rules list: add titles to icon buttons
- Logo and logo-mobile: change png to svg
- Roles page: padding, pointer
- Users page: bring in line with roles and realms: margins, button styling

showOkCancelDialog + text for ok action:
- or-mwc-dialog: showOkCancelDialog change button order and add option to change ok button text (to make action abundantly clear)

Edit mode header changes:
- Asset viewer: Sentence case for group panel attribute select dialog, **move and change edit button**, make edit header more distinctive, margin/padding

Dialogs responsive to size:
- Add attribute dialog: make size responsive to input fields, styling/spacing, **fix for [][]**
- or-rule-vality: make rule scheduler react to options

Comfortable and resizeVertical addition:
- or-mwc-input: **introduce comfortable(height:48px) and resizeVertical(textarea vertical resize)**, remove ripple for outlined fields
- or-attribute-input: introduce comfortable(height:48px) and resizeVertical(textarea vertical resize), spacing send button
- Edit asset panel: make input fields comfortable, make textareas resizable, remove borders from add attribute since it is not scrollable
- or-map: introduce comfortable for input field height

More customization for asset tree header:
- Asset tree + Add asset dialog: Add showDeselect option for parent select in Add asset dialog, add titles to icon buttons

Other:
- ValueFormat: fix numbers

### CODE REVIEW

File: ui/component/or-rules/src/or-rule-list.ts [hunk 1]
```diff
@@ -286,13 +286,13 @@ export class OrRuleList extends translate(i18next)(LitElement) {
...
+                        <or-mwc-input ?hidden="${this._isReadonly() || !this.selectedIds || this.selectedIds.length === 0}" type="${InputType.BUTTON}" icon="content-copy" @click="${() => this._onCopyClicked()}" title="Copy rule"></or-mwc-input>
+                        <or-mwc-input ?hidden="${this._isReadonly() || !this.selectedIds || this.selectedIds.length === 0}" type="${InputType.BUTTON}" icon="delete" @click="${() => this._onDeleteClicked()}" title="Delete rule"></or-mwc-input>
                         ${addTemplate}
-                        <or-mwc-input hidden type="${InputType.BUTTON}" icon="magnify" @click="${() => this._onSearchClicked()}"></or-mwc-input>
+                        <or-mwc-input hidden type="${InputType.BUTTON}" icon="magnify" @click="${() => this._onSearchClicked()}" title="Search"></or-mwc-input>
                         
                         ${getContentWithMenuTemplate(
-            html`<or-mwc-input type="${InputType.BUTTON}" icon="sort-variant"></or-mwc-input>`,
+            html`<or-mwc-input type="${InputType.BUTTON}" icon="sort-variant" title="Sort"></or-mwc-input>`,
```
Comment (richturner) [inline ui/component/or-rules/src/or-rule-list.ts:295]: `or-mwc-input` doesn't have a `title` property but probably should be added for consistency with standard `input` element...should then use `i18next.t("sort")

File: ui/app/manager/src/pages/page-users.ts [hunk 2]
```diff
@@ -395,10 +402,7 @@ class PageUsers<S extends AppStateKeyed> extends Page<S> {
                                     this._users = [...this._users, {enabled: true}];
                                 }}">
                                     <td colspan="100%">
-                                        <or-mwc-input class="button" .type="${InputType.BUTTON}"
-                                                      .label="${i18next.t("add")} ${i18next.t("user")}"
-                                                      icon="plus">
-                                        </or-mwc-input>
+                                        <a class="button"><or-icon icon="plus"></or-icon>${i18next.t("add")} ${i18next.t("user")}</a>
```
Comment (vlichtenberg) [inline ui/app/manager/src/pages/page-users.ts:405]: Some browsers might have a problem with `href` not being present in an anchor element. Check if `href="#"` works.

File: ui/app/manager/src/pages/page-users.ts [hunk 3]
```diff
@@ -438,10 +442,7 @@ class PageUsers<S extends AppStateKeyed> extends Page<S> {
                                         serviceAccount: true}]
                                 }}">
                                     <td colspan="100%">
-                                        <or-mwc-input class="button" .type="${InputType.BUTTON}"
-                                                      .label="${i18next.t("add")} ${i18next.t("user")}"
-                                                      icon="plus">
-                                        </or-mwc-input>
+                                        <a class="button"><or-icon icon="plus"></or-icon>${i18next.t("add")} ${i18next.t("user")}</a>
```
Comment (vlichtenberg) [inline ui/app/manager/src/pages/page-users.ts:445]: See https://github.com/openremote/openremote/pull/442/files#r662933008

File: ui/component/or-asset-viewer/src/or-add-attribute-panel.ts [hunk 4]
```diff
@@ -49,20 +49,18 @@ export class OrAddAttributePanel extends LitElement {
...
-            }
-
             or-mwc-input {
                 width: 300px;
                 display: block;
+                padding: 10px 20px;
             }
 
             .hidden {
-                visibility: hidden;
```
Comment (vlichtenberg) [inline ui/component/or-asset-viewer/src/or-add-attribute-panel.ts:65]: `visibility: hidden` can still be applied too.

File: ui/component/or-asset-viewer/src/style.ts [hunk 5]
```diff
@@ -87,19 +87,19 @@ export const style = css`
...
         overflow: auto;
+        min-height: calc(100vh - 88px - var(--internal-or-header-height));
     }
     
     #view-container {
         margin-top: 0;
         box-sizing: border-box;
         display: grid;
-        padding: 20px 20px;
+        padding: 0px 20px 20px;
```
Comment (vlichtenberg) [inline ui/component/or-asset-viewer/src/style.ts:99]: Extremely minor but `0px` in CSS can always be `0`.

File: ui/component/or-asset-viewer/src/style.ts [hunk 6]
```diff
@@ -150,13 +150,19 @@ export const style = css`
...
+        padding: 20px 30px;
         display: flex;
         flex: 0 0 auto;
         align-items: center;
         justify-content: space-between;
     }
 
+    #asset-header.editmode {
+        padding: 14px 30px;
+        background-color: white;
```
Comment (vlichtenberg) [inline ui/component/or-asset-viewer/src/style.ts:162]: Probably better to use one of the DefaultColor vars.

File: ui/component/or-rules/src/or-rule-validity.ts [hunk 7]
```diff
@@ -225,6 +225,30 @@ export class OrRuleValidity extends translate(i18next)(LitElement) {
...
+                    .mdc-dialog__surface {
+                        overflow-x: visible !important;
+                        overflow-y: visible !important;
+                    }
+
+                    #dialog-content {
+                        overflow: visible;
+                    }
+
+                    @media only screen and (max-width: 1279px) {
```
Comment (vlichtenberg) [inline ui/component/or-rules/src/or-rule-validity.ts:239]: Usually breakpoints in this area are at 1200. See for instance Bootstrap's media queries:
https://getbootstrap.com/docs/5.0/layout/breakpoints/#max-width

File: ui/component/or-rules/src/or-rule-list.ts (hunk 1 above)
Comment (DonWillems) [reply ui/component/or-rules/src/or-rule-list.ts]: Removed title properties for now. Want to add MDC tooltip to provide the desired result, while also being in line with accessibility guidelines. Added to backlog.

**Review by richturner — APPROVED** [review]

**Pull request merged**